docs/technical-documentation/identifiers_rework_design_decisions.md

```diff
@@ -0,0 +1,81 @@
+
+# Identifiers rework
```

Comment:
Hmm, in my [PR](https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3009/files#diff-04be0ef3d8e3b52ea5841470df7d55416b1534f74bcd8e167f09d2500848f28d), I created a stub of identifiers docs in `guides`, so that's available in registry site directly. I know that we also have other dosc in `technical-documentation`, but we should think if we should maybe move it/merge it?

Reply:
Yeah, I think there's a ticket, or we had multiple discussions about where we want to put the documentation to make it more available; putting it in the registry documentation would probably be a lot better than having it only in the repository. Maybe that should be a topic for one of our ✨ special Fridays ✨?